_posts/2018-12-11-grpc-stacks.md

```diff
@@ -9,21 +9,21 @@ company: Google
 company-link: https://www.google.com
 ---
 
-Here is a high level overview of the gRPC Stacks.  Each of the **10** default languages supported by gRPC has multiple layers, allowing you to customize what pieces you want in your application.
+Here is a high level overview of the gRPC Language Stacks.  Each of the **10** default languages supported by gRPC has multiple layers, allowing you to customize what pieces you want in your application.
 
 <!--more-->
 
-There are three main stacks in gRPC: C-core, Go, and Java.  Most of the languages are thin wrappers on top of the [C-based](https://github.com/grpc/grpc/tree/master/src/core) gRPC core library:
+There are three main language stacks in gRPC: C-core, Go, and Java.  Most of the languages are thin wrappers on top of the [C-based](https://github.com/grpc/grpc/tree/master/src/core) gRPC core library:
```

Comment:
nit: maybe something like "most of the user-facing language surface APIs", instead of "most of the languages". Otherwise, it's maybe a little confusing since "language stacks" including C-core, and then most "languages" are built on C-core